test/jsonapi/plugs/format_required_test.exs

```diff
@@ -136,6 +136,291 @@ defmodule JSONAPI.FormatRequiredTest do
     refute conn.halted
   end
 
+  test "halts and returns an error for a specified relationships param missing an object value" do
```

Comment:
I copy/pasted a lot of the tests below and just swapped in POST vs PATCH. Looks like I missed one in my haste.

Happy to do that here but I'll wait for some review feedback in case reviewers prefer `describe` blocks or tests that check both POST and PATCH in one case or something.

If the format used elsewhere below is fine I can add a PATCH case here also.

Reply:
Personally I would be inclined to use property based testing to cover POST and PATCH in one test case, but that's not going to block approval of this PR -- you can take your pick.

Reply:
Added the PATCH case.